.ci/pytorch/test.sh

```diff
@@ -255,6 +255,25 @@ test_inductor() {
   python test/run_test.py --include inductor/test_torchinductor inductor/test_torchinductor_opinfo --verbose
 }
 
+# "Global" flags for inductor benchmarking controlled by TEST_CONFIG
```

Comment:
I like how this makes everything cleaner. Since now we put more meaning to the naming of TEST_CONFIG, it may be worth to put some examples as comments here for explanation.